src/scripts/main.js

```diff
@@ -1,3 +1,200 @@
+      active.classList.remove('active');
+    });
+
+    element.classList.add('active');
+  });
+});
+
+const createForm = () => {
+  const newForm = document.createElement('form');
+
+  newForm.classList.add('new-employee-form');
+  /* eslint-disable max-len */
+
+  newForm.innerHTML = (
+    `<label>Name: <input name="name" type="text" data-qa="name" required></label>
+    <label>Position: <input name="position" type="text" data-qa="position" required></label>
+    <label>Office: <select name='office' data-qa="office" required><option value="Tokyo">Tokyo</option><option value="Singapore">Singapore</option><option value="London">London</option><option value="New York">New York</option><option value="Edinburgh">Edinburgh</option><option value="San Francisco">San Francisco</option></select></label>
+    <label>Age: <input name="age" type="number" data-qa="age" required></label>
+    <label>Salary: <input name="salary" type="number" data-qa="salary" required></label>
+    <button type='button' class='add-to-table'>Save to table</button>`
+  );
+
+  document.body.appendChild(newForm);
+};
+
+createForm();
+
+const addButton = document.querySelector('.add-to-table');
+const addForm = document.querySelector('.new-employee-form');
+
+addButton.addEventListener('click', () => {
+  const formValues = {
+    Name: addForm.querySelector('input[name="name"]').value,
+    Position: addForm.querySelector('input[name="position"]').value,
+    Office: addForm.querySelector('select[name="office"]').value,
+    Age: Number(addForm.querySelector('input[name="age"]').value),
+    Salary: Number(addForm.querySelector('input[name="salary"]').value),
+  };
+
+  if (formValues.Name.length < 4) {
+    pushNotification(10, 10, `Wrong name: ${formValues.Name}`, 'Your name is to short', 'error');
+  } else if (formValues.Age < 18 || formValues.Age > 90) {
+    pushNotification(10, 10, `Wrong Age: ${formValues.Age}`, 'Your age should be between 18 - 90', 'error');
+  } else if (Object.values(formValues).every(value => String(value).length > 0)) {
+    users.push(formValues);
+    createTable(users);
+    pushNotification(10, 10, `Success`, 'New employee added to table', 'success');
+  } else {
+    pushNotification(10, 10, `Error`, 'all inputs must be filled', 'error');
+  }
+});
+
+const pushNotification = (posTop, posRight, title, description, type) => {
+  const notification = document.createElement('div');
+
+  notification.classList.add(`notification`);
+  notification.classList.add(type);
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.innerHTML = (
+    `<h2 class='title'>${title}</h2>
+    <p>${description}</p>`
+  );
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    document.body.removeChild(notification);
+  }, 2000);
+};
+
+let currentValue;
+
+tableBody.addEventListener('click', (clickEvent) => {
```

Comment:
Consider using event delegation for handling the editing of table cells, instead of adding event listeners for each cell.